src/utils/fuzzySearch.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @flow
+ * @file Basic fuzzy search algorithm. Matches all characters in search string to content in the order they appear.
+ * @author Box
+ *
+ * @param {string} search User input search string
+ * @param {string} content Content to search over for matches
+ * @param {number} minCharacters Minimum number of search characters before matching anything, default 3
+ * @param {number} maxGaps Approximate maximum number of gaps in the search string to tune fuzzyness, default 2
+ * @returns {boolean} If a match is found
+ */
+const fuzzySearch = (search: string, content: string, minCharacters: number = 3, maxGaps: number = 2) => {
+    content = content.toLowerCase().replace(/\s/g, '');
```

Comment:
Although our eslint config allows it and these are pass-by-value, I think we usually avoid param reassignment in most cases. Can we assign these to some new local variables?